src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -469,7 +471,7 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
         try {
           const graph = provider.graph.forComponent(this);
           this._groupPeople = await getPeopleFromGroup(graph, this.groupId);
-        } catch {
+        } catch (_) {
```

Comment:
Not familiar with this... what does this do for Edge/IE?

Reply:
Edge is complaining with this:

```js
try {
...
} catch { // exception in Edge, expect "(" here
...
}
```

The fix here is to just add a dummy catch statement. `_` is usually used for those purposes